.changes/unreleased/NOTES-20240229-144205.yaml

```diff
@@ -0,0 +1,7 @@
+kind: NOTES
+body: 'all: This Go module has been updated to Go 1.21 per the [Go support policy](https://go.dev/doc/devel/release#policy).
```

Comment:
nit: Similar question to the framework PR, should we mention that `v0.15.0` introduced Go 1.21 to this module?

Reply:
Have updated the changelog